hotsos/core/plugins/sosreport.py

```diff
@@ -31,16 +31,20 @@ def data_root_is_sosreport(self):
     @cached_property
     def version(self):
         if not self.data_root_is_sosreport:
-            return
+            return None
 
         path = os.path.join(HotSOSConfig.data_root, 'version.txt')
         if not os.path.exists(path):
-            return
+            return None
 
         with open(path) as fd:
             for line in fd:
-                if line.startswith('sosreport:'):
-                    return line.partition(' ')[2].strip()
+                if not line.startswith('sosreport:'):
```

Comment:
In future, one might very well think the LHS is better and rewrite that way again ;-) (except the final `return None` of course).